R/anova_tidiers.R

```diff
@@ -103,22 +103,28 @@ tidy.aov <- function(x, ...) {
 
 
 #' @rdname anova_tidiers
+#' 
+#' @importFrom plyr ldply
+#' 
+#' @import dplyr
```

Comment:
`dplyr` and `plyr` get imported package-wide in DESCRIPTION, so I believe these aren't necessary.

Reply:
Got it. I'll remove them.